### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by frenchi**
Update reference to current draft PR for Project Zenith from #5443 (now closed) to #5757 (current WIP draft PR)

### CODE REVIEW

**Comment by frenchi** [comment]
failed check: docs/readthedocs.org:dagger-io

seems unrelated.

https://readthedocs.org/projects/dagger-io/builds/22182852/

```
Theme error:
no theme named 'sphinx_rtd_theme' found (missing theme.conf?)
```

**Comment by sipsma** [comment]
> I ran into this in various other PRs as well. Could you please take a look? Thanks.

I fixed this in a recent merge to main, it should be fixed once this PR is rebased.

File: zenith/README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ As of August 24 2023, the latest source code for Project Zenith is available as
 
 To get started:
 
-1. Visit the [draft pull request](https://github.com/dagger/dagger/pull/5443)
+1. Visit the [current draft pull request](https://github.com/dagger/dagger/pull/5757) and the [original draft PR (now closed)](https://github.com/dagger/dagger/pull/5443)
 2. Checkout the corresponding branch
```
Comment (sipsma) [inline zenith/README.md:35]: We actually merged that PR too as of a few hours ago 😄 So I actually think we can just remove `1.` and `2.` from here now.

**Review by sipsma — COMMENTED** [review]
One quick comment, thank you for the fix here!

File: zenith/README.md (hunk 1 above)
Comment (frenchi) [reply zenith/README.md:35]: Moving fast!! I'll close this PR then, as it seems like the readme has moved on.

**Pull request closed without merging**